components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -101,6 +102,7 @@
 
 <ncdata dyn="se" hgrid="ne0np4_antarcticax4v1" nlev="72"    >atm/cam/inic/homme/cami_mam3_Linoz_0000-01-antarcticax4v1_L72_topoadj_c20200507.nc</ncdata>
 
+<ncdata dyn="se" hgrid="ne0np4_arcticx4v1" nlev="72"    >atm/cam/inic/homme/cami_mam3_Linoz_0000-01-arcticx4v1np4_L72_c20210429.nc</ncdata>
```

Comment:
This IC was a remapping from ne120np4; no sign of topo adjustment. Has it been tested that the model can run stably with it?